jujugui/static/gui/src/app/components/unit-list/test-unit-list.js

```diff
@@ -49,10 +49,12 @@ describe('UnitList', () => {
       <juju.components.UnitListItem
         key={units[0].displayName}
         label={units[0].displayName}
+        action={output.props.children[1].props.children[1].props.action}
```

Comment:
This is a little trick I found to be able to write test assertions using a component output as the comparison. The issue we found was that these functions are internally bound to so there is no way to pass in a function which matches what this should be. Instead we reach deep into the React object and grab the function we expect to be there and use it to compare against.